torch_geometric/utils/augmentation.py

```diff
@@ -0,0 +1,204 @@
+from typing import Optional, Tuple, Union
+
+import torch
+from torch import Tensor
+
+from .num_nodes import maybe_num_nodes
+
+
+def shuffle_node(x: Tensor, training: bool = True) -> Tuple[Tensor, Tensor]:
```

Comment:
Should we support the `shuffle_node` for batched graphs?

Reply:
That's a great idea. Let's add it as a TODO.

Reply:
Sounds great! I think I can add it later.